.gitignore

```diff
@@ -0,0 +1,6 @@
+*.egg-info
```

Comment:
I understand the argument, but that might be relying too much on local set-up. Not everyone will have their global git config setup that way.